### PR TITLE
Help Tickets: Add confirmation button

### PR DESCRIPTION
**Pull request by HoeenCoder**
Adds a button that you need to click to confirm you want to open a help ticket before its created.

### CODE REVIEW

File: chat-plugins/helptickets.js [hunk 1]
```diff
@@ -299,22 +330,26 @@ exports.pages = {
 					buf += `<p><Button>timerstalling</Button></p>`;
 					break;
 				case 'harassment':
+					if (!isLast) break;
 					buf += `<p>If someone is harassing you, click the appropriate button below and a global staff member will take a look. Consider using <code>/ignore [username]</code> if it's minor instead.</p>`;
 					buf += `<p>If you are reporting harassment in a battle, please save a replay of the battle.</p>`;
-					buf += `<p><button class="button" name="send" value="/helpticket submit PM Harassment">Report harassment in a private message (PM)</button> <button class="button" name="send" value="/helpticket submit Battle Harassment">Report harassment in a battle</button> <button class="button" name="send" value="/helpticket submit Chatroom Harassment">Report harassment in a chatroom</button></p>`;
+					buf += `<p><Button>confirmpmharrasment</Button> <Button>confirmbattleharrasment</Button> <Button>confirmchatharrasment</Button></p>`;
```
Comment (panpawn) [inline chat-plugins/helptickets.js]: ``<button>``

Also did you mean to make these buttons not do anything?

Comment (panpawn) [reply chat-plugins/helptickets.js]: (sane thing later down, too)

Comment (HoeenCoder) [reply chat-plugins/helptickets.js]: Theres a regexp at the bottom that transforms them, hence why its `<Button>`

Comment (panpawn) [reply chat-plugins/helptickets.js]: Yeah, maybe this should use a new different tag though

**Comment by HoeenCoder** [comment]
Build is not running for some reason, It did pass when i ran `npm test` though locally.

File: chat-plugins/helptickets.js (hunk 1 above)
Comment (Zarel) [reply chat-plugins/helptickets.js]: It does, it uses a capital B. :|

File: chat-plugins/helptickets.js [hunk 2]
```diff
@@ -299,22 +330,26 @@ exports.pages = {
 					buf += `<p><Button>timerstalling</Button></p>`;
 					break;
 				case 'harassment':
+					if (!isLast) break;
```
Comment (Zarel) [inline chat-plugins/helptickets.js]: You break _after_ the text and before the buttons.

File: chat-plugins/helptickets.js [hunk 3]
```diff
@@ -364,16 +402,23 @@ exports.pages = {
...
 					break;
 				case 'other':
+					if (!isLast) break;
 					buf += `<p>If your issue is not handled above, click the button below to ask for a global. Please be ready to explain the situation.</p>`;
-					buf += `<p><button class="button" name="send" value="/helpticket submit Other">Call a global staff member</button></p>`;
+					buf += `<p><Button>confirmother</Button></p>`;
+					break;
+				default:
+					if (!page.startsWith('confirm')) break;
+					buf += `<p><b>Are you sure you want to submit a report? Type: ${ticketTitles[page.slice(7)]}</b></p>`;
```
Comment (Zarel) [inline chat-plugins/helptickets.js]: This is vague, "Type:" makes me think "You should type the following text with your keyboard:"

Maybe `submit a "${ticketTitles[page.slice(7)]}" report?`

**Pull request merged**